db/migrate/20151106182639_change_project_attributes_public_default.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeProjectAttributesPublicDefault < ActiveRecord::Migration
+  def change
+    rename_column :project_attributes, :hidden, :public
+    change_column_default :project_attributes, :public, true
```

Comment:
I believe this migration need to flip the values for existing rows, but it doesn't seem to be doing it.